src/services/pocket-relayer.ts

```diff
@@ -848,14 +821,40 @@ export class PocketRelayer {
     )
   }
 
-  static async stickyRelayResult(
-    preferredNodeAddress: string | undefined,
-    relayNodePublicKey: string
-  ): Promise<string> {
-    if (!preferredNodeAddress) {
-      return 'NONE'
+  async setStickinessKey(
+    { duration, keyPrefix, rpcID }: StickinessOptions,
+    blockchainID: string,
+    applicationID: string,
+    nodeAddress: string,
+    data: string
+  ): Promise<void> {
+    if (!keyPrefix && !rpcID) {
+      return
     }
 
-    return preferredNodeAddress === (await getAddressFromPublicKey(relayNodePublicKey)) ? 'SUCCESS' : 'FAILURE'
+    // If no key prefix is given, set based on rpcID
```

Comment:
Better comment here too please -- explain why there might be a key prefix and what it might be